frontend/src/components/7.Result/ResultPage.js

```diff
@@ -21,7 +21,7 @@ const rating = '4.0';
  * @return {*}
  * TODO: remove hard-coded location for the winning restaurant coordinates
  */
-function ResultPage(props) {
```

Comment:
Was the use of props here because of the code used? before the use of sockets or something

Reply:
I'm not entirely sure @David-Xia0 added it in I think. But I decided to remove it since it wasn't being used.